<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addItemBtn">
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="2">
      <layout class="QFormLayout" name="statsLayout"/>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="inventoryList"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="setItemPropertiesBtn">
       <property name="text">
        <string>SET</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="removeItemBtn">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="addRandItemBtn">
     <property name="text">
      <string>Add Random</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
